eng/scripts/Language-Settings.ps1

```diff
@@ -6,25 +6,32 @@ $packagePattern = "*.tgz"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/js-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=javascript%2F&delimiter=%2F"
 
+if ((Get-Command npm | Measure-Object).Count -eq 0 ) 
```

Comment:
Why add this here as part of this change? I'm not opposed to it but it seems a little out of place.

Reply:
Well, I wanted to do a single check for node, as suggested by @mitchdenny

Reply:
There are lots of things in this script that will not require node so we should wrap this in a function and call it only in the places where it is required.